docs/blog/2022-05-02-arch-linux-installation-guide.mdx

```diff
@@ -0,0 +1,355 @@
+---
+title: "Arch Linux Installation Guide For Developers"
+description: We sharing our process of installing Arch Linux for developers the way we do it in house
+slug: arch-linux-installation-guide
+date: 2022-05-04
+image: https://www.lunasec.io/docs/img/clippy_arch.png
+keywords: [arch, arch-linux, install-guides]
+tags: [developer-tools, install-guides, archlinux, arch]
+authors: [gabe]
+
+---
+<!--
+  ~ Copyright by LunaSec (owned by Refinery Labs, Inc)
+  ~
+  ~ Licensed under the Creative Commons Attribution-ShareAlike 4.0 International
+  ~ (the "License"); you may not use this file except in compliance with the
+  ~ License. You may obtain a copy of the License at
+  ~
+  ~ https://creativecommons.org/licenses/by-sa/4.0/legalcode
+  ~
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+-->
+![Arch Linux Install](https://www.lunasec.io/docs/img/clippy_arch.png)
+
+## BTW we use Arch
+
+Here at LunaSec we've been busy hacking away at some exciting products, we've also onboarded a couple new team members WOOHOO 🥳! Through the onboarding process we realized there was a great opportunity to
+release our developer environment setup as a guide for the community.
+
+We hope this guide helps encourage anyone
+looking to start their journey into the world of Arch Linux and provides confidence to any teams considering Arch as a potential development environment.
+
+
+## Why Arch is the best operating system for developers?
+Here's a few of the reasons why decided to use Arch as our development environment
+- We have direct visibility and control down to the kernel
+- We know what's on our system and only use what we deem necessary
+- We can configure our environments to meet security standards quickly and seamlessly
+- We have access to the latest packages for development
+
+We found arch to be that sweet spot between, security, accessibility and culture. For us using secure, open source software
+that is constantly updated is our baseline. We found that Arch was the best choice for meeting each of these requirements.
+
```

Comment:
I think we need another paragraph here that quickly explains that we will be using full disk encryption, a somewhat experimental filesystem, and systemdboot.  We should probably say WHAT we are installing before just jumping into it, because this isnt really a standard arch install

Reply:
"A security professional's arch installation, with the latest and greatest options"

Reply:
hmmm, yeah tht makes sense, I can make that more explicit.